### PR TITLE
Update get-partitionsupportedsize.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#982 
Action Taken:
Added "This command will show the actual value to see the minimum and maximum size for the partition (SizeMin and SizeMax)."

See screenshot: https://snag.gy/GjqpDB.jpg

See article:
https://docs.microsoft.com/en-us/powershell/module/storage/get-partitionsupportedsize?view=win10-ps#feedback - not updated
still show up as: "MinimumSize"  "MaximumSize "

See example in this article as well:
https://docs.microsoft.com/en-us/powershell/module/storage/resize-partition?view=winserver2012r2-ps

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/storage/get-partitionsupportedsize.md [hunk 1]
```diff
@@ -65,8 +65,7 @@ PS C:\>Get-PartitionSupportedSize -DiskNumber 3 -PartitionNumber 2
 SizeMin                                                 SizeMax 
 -----------                                             -----------
 ```
-
-This example is not fully implemented and does not include size values.
+This command will show the actual value to see the minimum and maximum size for the partition (SizeMin and SizeMax).
```
Comment (JohanFreelancer9) [inline docset/windows/storage/get-partitionsupportedsize.md]:
```suggestion
This command will show the actual minimum and maximum size of the partition (SizeMin and SizeMax).
```

**Comment by get-itips** [comment]
I accept the additions after @JohanFreelancer9 edit, but I am not seeing in the PR the original issue fixed where the user says to correct Get-PartitionSupportedSize**s** in the Description.

**Comment by dariomws306** [comment]
I am agree with this PR.

@get-itips looks like @sowjanya365 removed on Apr 24 the 's'.

**Comment by get-itips** [comment]
@dariomws306 oh thanks for the clarification.

**Comment by tseward** [comment]
I'd drop the "actual" but am otherwise OK with this.

**Comment by e0i** [comment]
@AndreaBarr This PR can be merged. Thanks.

**Pull request merged**